data_Insights_website/static/css/style.css

```diff
@@ -1,331 +1,301 @@
+  text-align: center;
+  color: white; }
 
 .card-desc-small {
-    background-color: rgba(46, 46, 46, 0.3);
-    width: 255px;
-    height: 340px;
-    margin: 0 4rem;
-    display: inline-block;
-}
-
-.card-desc-small .card-desc-small-text {
+  background-color: rgba(46, 46, 46, 0.3);
+  width: 255px;
+  height: 340px;
+  margin: 0 4rem;
+  display: inline-block; }
+  .card-desc-small .card-desc-small-text {
     font-family: "quicksand", sans-serif;
-    padding: 1rem 2rem;
-}
+    padding: 1rem 2rem; }
 
 .card-desc-small .card-desc-small-img {
-    height: 160px;
-    line-height: 160px;
-    background-repeat: no-repeat;
-    background-size: cover;
-}
+  height: 160px;
+  line-height: 160px;
+  background-repeat: no-repeat;
+  background-size: cover; }
 
 .card-lg {
-    background-color: rgba(46, 46, 46, 0.3);
-    width: 100%;
-    margin: 0 4rem;
-    display: inline-block;
-}
-
-.card-lg .card-lg-text {
+  background-color: rgba(46, 46, 46, 0.3);
+  width: 100%;
+  margin: 0 4rem;
+  display: inline-block; }
+  .card-lg .card-lg-text {
     font-family: "quicksand", sans-serif;
-    padding: 1rem 2rem;
-}
+    padding: 1rem 2rem; }
+
+li.list-group-item {
+  background: rgba(0, 0, 0, 0) !important;
+  border-bottom: 0.15rem solid rgba(255, 255, 255, 0.5);
+  line-height: 4rem;
+  font-size: 1.25rem;
+  cursor: pointer; }
+
+li.list-group-item > span {
+  color: white;
+  display: flex;
+  padding-left: 0.2rem; }
+
+li.list-group-item > i {
+  position: absolute;
+  right: 1.3rem;
+  top: 42%; }
+
+ul.list-group > li:first-child {
+  border-top: 0.15rem solid rgba(255, 255, 255, 0.5); }
+
+.dropdown {
+  position: relative;
+  display: inline-block; }
+
+/* Dropdown Content (Hidden by Default) */
+.dropdown-content {
+  display: none;
+  position: relative;
+  background-color: #2e2e2e;
+  min-width: 5rem;
+  max-width: 16.5rem !important;
+  box-sizing: border-box;
+  left: 2rem; }
+
+/* put borders on the top and bottom of  all the list link elements*/
+div.dropdown-content > a:first-child {
+  border-top: 0.15rem solid rgba(255, 255, 255, 0.5); }
+
+div.dropdown-content > a {
+  border-bottom: 0.15rem solid rgba(255, 255, 255, 0.5); }
+
+/* styling the dropdown contents by inheriting from ms-text */
+div.dropdown-content > a.ms-text {
+  line-height: 3rem !important;
+  padding-top: 1rem; }
+
+/* Links inside the dropdown */
+.dropdown-content a {
+  color: white;
+  text-decoration: none;
+  display: block; }
+
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {
+  background-color: black; }
+
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {
+  display: block; }
 
 .carousel-indicators li {
-    width: 7px;
-    height: 7px;
-    border-radius: 100%;
-}
+  width: 7px;
+  height: 7px;
+  border-radius: 100%; }
 
 .carousel-indicators {
-    bottom: -35px;
-}
+  bottom: -35px; }
 
 #quotes-carousel {
-    width: 750px;
-    margin: auto;
-}
+  width: 750px;
+  margin: auto; }
```

Comment:
You forgot to run beautify this css file, so this commit is coming to undo the previous one

Reply:
You should probably delete the previous commit and create a new one where you format this file appropriately